muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/sass/elements/course-description.scss

```diff
@@ -0,0 +1,21 @@
+// Everything in here only affects coursepicker's course description
+// and has to be nested as everything styled here are created by CKEditor
+// so we have no say what kind of html content we have in course descrition 
+
```

Comment:
I'm wondering, if this is a more general thing. As of now, of course there are no other cases of mainfunction descriptions.